templates/account.html

```diff
@@ -43,22 +43,32 @@ <h3>My snaps</h3>
                   {% else %}
                     {% for snap in user_snaps|sort %}
                         <tr>
-                            <td class="u-vertically-center">
-                                {% if user_snaps[snap].icon_url %}
-                                    <img src="{{ user_snaps[snap].icon_url }}" width="32" height="32" />&nbsp;
-                                {% else %}
-                                    <img src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" width="32" height="32" />&nbsp;
-                                {% endif %}
-                                {% if user_snaps[snap].uploaded %}
-                                    <a href="/account/snaps/{{snap}}/market" class="u-no-margin--top">{{snap}}</a>
+                            <td class="u-vertically-center p-snap-list">
+                              {% if user_snaps[snap].uploaded %}
+                                <a href="/account/snaps/{{snap}}/market" class="u-no-margin--top u-vertically-center">
+                              {% endif %}
+                              {% if user_snaps[snap].icon_url %}
+                                  <img src="{{ user_snaps[snap].icon_url }}" width="32" height="32" class="p-snap-list__image" />
+                              {% else %}
+                                  <img src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" width="32" height="32" class="p-snap-list__image" />
+                              {% endif %}
+                              <span class="p-snap-list__name u-no-margin--top">
+                                  {{snap}}
+                              </span>
+                              {% if user_snaps[snap].uploaded %}
+                                </a>
+                              {% endif %}
+                              {% if user_snaps[snap].private %}
+                                  <small class="u-no-margin--top">Private</small>
+                              {% endif %}
+                            </td>
+                            <td>
+                                {% if not user_snaps[snap].uploaded %}
+                                    Name registered
```

Comment:
Don't we want a message like: `Snap not uploaded`?

Reply:
Additionally to a better `status` terminology, I would also order the list by their status, then name.  Snaps 'uploaded' would be listed first.
Later on we can introduce new (default) order, possibly by last upload/change or installed_base.

Reply:
@cprov new issue created here: https://github.com/canonical-websites/snapcraft.io/issues/323 /cc @tbille